api/test/wfl/integration/modules/covid_test.clj

```diff
@@ -12,6 +13,77 @@
            [java.time OffsetDateTime]
            [java.util UUID]))
 
+(clj-test/use-fixtures :once fixtures/temporary-postgresql-database)
```

Comment:
Remove the `clj-test/` namespace alias since we've `:require`d `:refer :all` above.

Reply:
Done